Clinical-Genomics/cgweb
# Order rna

These PR's adds the possibility to order MIP RNA in the OP

**How to test**:
1. install on stage of hasta `bash servers/resources/hasta.scilifelab.se/update-cg-stage.sh order-rna`
1. install on stage of clinical-db `bash servers/resources/hasta.scilifelab.se/update-cgweb-stage.sh order-rna`

For tests see Validation Report - MIP RNA orders in the order portal (cg and cgweb) in AM


**Review:**
- [x] code approved by @ingkebil 
Thanks for filling in who performed the code review and the test!

Part of the RNA validation ams#2016
